skimage/__init__.py

```diff
@@ -1,68 +1,53 @@
 """Image Processing for Python
 
-``scikit-image`` (a.k.a. ``skimage``) is a collection of algorithms for image
+scikit-image (a.k.a. ``skimage``) is a collection of algorithms for image
 processing and computer vision.
 
 The main package of ``skimage`` only provides a few utilities for converting
-between image data types; for most features, you need to import one of the
-following subpackages:
+between image data types; for most features, you need to import one of its
+subpackages.
```

Comment:
I just realized this is no longer true! It is now *use* one of its subpackages, since they are directly accessible.

Reply:
Oh, right!! And the first part of this sentence doesn't hold anymore either, since we're moving away from accessing the `img_as_float` functions from the root package. I would remove this sentence altogether.